rest_framework_json_api/renderers.py

```diff
@@ -396,7 +396,7 @@ def build_json_resource_obj(cls, fields, resource, resource_instance, resource_n
             resource_data.append(('links', {'self': resource[api_settings.URL_FIELD_NAME]}))
         return OrderedDict(resource_data)
 
-    def render_relationship_view(self, data, accepted_media_type=None, renderer_context=None):
+    def render_relationship_view(self, data, accepted_media_type=None, renderer_context={}):
```

Comment:
Does this mean you're ok closing this PR? This comment makes it seem to me like this is not important to you anymore, but I might be misreading. I'm wondering if I should spend the time to review or not.